docs/samples/v1beta1/transformer/feast/driver_transformer/driver_transformer/driver_transformer.py

```diff
@@ -101,20 +100,26 @@ def buildPredictRequest(self, inputs, features) -> Dict:
 
         """
         request_data = []
-        for i in range(len(features["field_values"])):
-            entity_req = [features["field_values"][i]["fields"][self.feature_refs_key[j]]
-                          for j in range(len(self.feature_refs_key))]
-            for j in range(len(self.entity_ids)):
-                entity_req.append(features["field_values"][i]["fields"][self.entity_ids[j]])
-                request_data.insert(i, entity_req)
+        acc_rate_index = features["metadata"]["feature_names"].index("driver_hourly_stats__acc_rate")
+        avg_daily_trips_index = features["metadata"]["feature_names"].index("driver_hourly_stats__avg_daily_trips")
+        conv_rate_index = features["metadata"]["feature_names"].index("driver_hourly_stats__conv_rate")
+        entity_ids_index = features["metadata"]["feature_names"].index("driver_id")
+
+        # input format [acc_rate, avg_daily_trips, conv_rate, driver_id]
+        for i in range(len(features["results"][entity_ids_index]["values"])):
+            single_entity_data = [features["results"][acc_rate_index]["values"][i],
+                                  features["results"][avg_daily_trips_index]["values"][i],
+                                  features["results"][conv_rate_index]["values"][i],
+                                  features["results"][entity_ids_index]["values"][i]]
+            request_data.append(single_entity_data)
 
         # The default protocol is v1
         result = {'instances': request_data}
         if self.protocol == "v2":
```

Comment:
@sivanantha321 Can you help update this code with the latest 0.10 changes that you no longer need to handle rest/gRPC separately, if you return the unified type `InferRequest` then kserve can translate accordingly.

see example here https://github.com/kserve/kserve/blob/master/python/custom_transformer/model.py#L74

Reply:
sure